index.js

```diff
@@ -64,7 +61,6 @@ module.exports = function (babel, args) {
     config.plugins.push(['babel-plugin-css-modules-transform', {
       generateScopedName: '[name]__[local]___[hash:base64:5]',
     }]);
-    config.plugins.push(['@babel/plugin-proposal-object-rest-spread', { useBuiltIns: true }]);
```

Comment:
This used to be "at the end" of plugins, now it's at the beginning, right? Might be worth putting it at the end to minimize unintended consequences.

Reply:
Actually what was happening boils down to
```javascript
if (isWebpack) {
  config.plugins.unshift(['@babel/plugin-proposal-object-rest-spread', { useBuiltIns: true }]);
} else {
  config.plugins.push(['@babel/plugin-proposal-object-rest-spread', { useBuiltIns: true }]);
}
```
but I don't believe that it matters if it's at the end or the beginning. Do you know @jasisk?

Reply:
I also have no reason to believe it, but this is an area I try not to poke the beast.

Reply:
Actually, the other point is that I'm not sure it's NEEDED on the node side anymore.

Reply:
You are correct, `@babel/plugin-proposal-object-rest-spread` is included in `babel-preset-env`. I will remove it and verify on one of our projects that it works as expected.